### PR TITLE
PXB-3354 Add glibc information to tarball topic

### PR DESCRIPTION
**Pull request by patrickbirch**
On branch pxb-3354
	modified:   docs/binary-tarball-names.md

### CODE REVIEW

File: docs/binary-tarball-names.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+The following lists the platform and the associated full binary file name used by Percona Server for MySQL tar files {{release}}.
+
+| Platform             | Percona XtraBackup tarball name                   | glibc2 version |
+|----------------------|--------------------------------------------------------------|-----------|
+| Ubuntu 24.04         | Percona-XtraBackup-{{release}}-Linux.x86_64.glibc2.39.tar.gz     | glibc2.39 |
+| Ubuntu 22.04         | Percona-XtraBackup-{{release}}-Linux.x86_64.glibc2.35.tar.gz     | glibc2.35 |
+| Ubuntu 20.04         | Percona-XtraBackup-{{release}}-Linux.x86_64.glibc2.31.tar.gz     | glibc2.31 |
+| Red Hat Enterprise 9 | Percona-XtraBackup-{{release}}-Linux.x86_64.glibc2.34.tar.gz     | glibc2.34 |
+| Red Hat Enterprise 8 | Percona-XtraBackup-{{release}}-Linux.x86_64.glibc2.28.tar.gz     | glibc2.28 |
+| Red Hat Enterprise 7 | Percona-XtraBackup-{{release}}-Linux.x86_64.glibc2.17.tar.gz     | glibc2.17 |
```
Comment (satya-bodapati) [inline docs/binary-tarball-names.md]: should be we expand it a bit to include debian, Centos, OEL names?

File: docs/binary-tarball-names.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+
+## Binary tarball file name organization
+
+The following lists the platform and the associated full binary file name used by Percona Server for MySQL tar files {{release}}.
+
+| Platform             | Percona XtraBackup tarball name                   | glibc2 version |
+|----------------------|--------------------------------------------------------------|-----------|
+| Ubuntu 24.04         | Percona-XtraBackup-{{release}}-Linux.x86_64.glibc2.39.tar.gz     | glibc2.39 |
+| Ubuntu 22.04         | Percona-XtraBackup-{{release}}-Linux.x86_64.glibc2.35.tar.gz     | glibc2.35 |
+| Ubuntu 20.04         | Percona-XtraBackup-{{release}}-Linux.x86_64.glibc2.31.tar.gz     | glibc2.31 |
```
Comment (satya-bodapati) [inline docs/binary-tarball-names.md:28]: 24.04/ Debian 13 (trixie)
22.04/Debian 12 (bookworm)
20.04/Debian 11 (bullseye)

File: docs/binary-tarball-names.md (hunk 1 above)
Comment (satya-bodapati) [inline docs/binary-tarball-names.md:33]: RHEL9/Centos9/OEL9
RHEL8/Centos8/OEL8
RHEL7/Centos7/OEL7

Comment (satya-bodapati) [reply docs/binary-tarball-names.md]: somone from QA/buildteam can help us to verify if these substitute names are correct

File: docs/binary-tarball-names.md (hunk 2 above)
Comment (patrickbirch) [reply docs/binary-tarball-names.md:28]: We don't use the nicknames

File: docs/binary-tarball-names.md (hunk 1 above)
Comment (patrickbirch) [reply docs/binary-tarball-names.md]: Shouldn't use CentOS.
I can add Debian. Other YUM-based OS are derivatives of RHEL.

File: docs/binary-tarball-names.md (hunk 2 above)
Comment (patrickbirch) [reply docs/binary-tarball-names.md:28]: Also Debian 13 is not released yet and is not expected until 2025.

File: docs/binary-tarball-names.md (hunk 1 above)
Comment (patrickbirch) [reply docs/binary-tarball-names.md:33]: The others are derivatives.

File: docs/binary-tarball-names.md [hunk 3]
```diff
@@ -1,5 +1,37 @@
...
+| Debian 12            | Percona-XtraBackup-{{release}}-Linux.x86_64.glibc2.36.tar.gz     | glibc2.36 |
+| Debian 11            | Percona-XtraBackup-{{release}}-Linux.x86_64.glibc2.31.tar.gz     | glibc2.31 |
+| Red Hat Enterprise 9 | Percona-XtraBackup-{{release}}-Linux.x86_64.glibc2.34.tar.gz     | glibc2.34 |
+| Red Hat Enterprise 8 | Percona-XtraBackup-{{release}}-Linux.x86_64.glibc2.28.tar.gz     | glibc2.28 |
+| Red Hat Enterprise 7 | Percona-XtraBackup-{{release}}-Linux.x86_64.glibc2.17.tar.gz     | glibc2.17 |
+
 Download the binary tarballs from [Percona Product Downloads].
```
Comment (alina-derkach-oaza) [inline docs/binary-tarball-names.md]: [Percona Product Downloads] -> [Percona Software Downloads]

**Review by alina-derkach-oaza — APPROVED** [review]

**Review by satya-bodapati — APPROVED** [review]
LGTM!

**Pull request merged**